### ISSUE TITLE
Resolve CodeQL alert 25

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#5642]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
We need to analyze [CodeQL query alert 25](https://github.com/hackforla/website/security/code-scanning/25) then either recommend dismissal of the alert or update the code to resolve the alert.   

### Action Items
- [x] DO NOT DISMISS ANY ALERTS.  Dismissal of alerts should be done by dev leads only after review of the recommendation
- [x] Browse to the link in the next Action Item and read the contents.  Click "See More" to view Recommendations, Examples and References.  
- [x] https://github.com/hackforla/website/security/code-scanning/25  
- [x] In a comment in this issue, add your analysis and recommendations.  The recommendation can be one of the following: `dismiss as test`, `dismiss as false positive`, `dismiss as won't fix`, or `update code`.  An example of a `false positive` is a report of a JavaScript syntax error that is caused by markdown or liquid symbols such as `---` or `{%`  
- [x] If the recommendation is to dismiss the alert, apply the label `ready for dev lead` then move the issue to `Questions/In Review`
- [x] If the recommendation is to update code:
   - [x] create an issue branch and proceed with the code update
   - [x] test using docker to ensure that there are no changes to any affected webpage(s)
   - [x] proceed with pull request in the usual manner 

### For merge team/dev lead
- [ ] If recommendation to dismiss is approved, dismiss the alert with a comment, then close the issue as completed.
- [ ] If recommendation to update code is approved, move the issue to "In Progress", remove "ready for dev lead" label and notify assignee to proceed
- [ ] In either case when this issue is closed please check off the dependency (under "Analysis Issues") in #5060.  If all analysis issues are closed, close #5060 as completed.  
  
### Resources/Instructions
- [GitHub CodeQL documentation](https://codeql.github.com/docs/codeql-overview/about-codeql/)
- This issue is part of #5060

**Comment by ExperimentsInHonesty** [comment]
@roslynwythe the link under 2 in the resources does not resolve.

**Comment by roslynwythe** [comment]
> ### Prerequisite
> 1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
> 2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).
> 
> ### Overview
> We need to analyze [CodeQL query alert 25](https://github.com/hackforla/website/security/code-scanning/25) and to recommend appropriate disposition of the alert. Options include dismissal of the alert or updating the code.
> 
> ### Action Items
> * [ ]  DO NOT DISMISS ANY ALERTS.  Dismissal of alerts should be done by dev leads only after review of the recommendation
> * [ ]  Browse to the link in the follosing Action Item and read the contents.  Click "See More" to view Recommendations, Examples and References.
> * [ ]  [Malformed id attribute](/hackforla/website/security/code-scanning/25)
> * [ ]  In a comment in this issue, add your analysis and recommendation.  Recommendation can be one of the following: `dismiss as test`, `dismiss as false positive`, `dismiss as won't fix`, or `update code`.  If the recommendation is `update code`, describe the code change in detail.  If the code change affects only 1-2 lines, show the existing and proposed code.
> * [ ]  Apply the label `ready for dev lead` then move the issue to `Questions/In Review`
> * [ ]  After review, if the recommendation is to fix code, proceed to make the code fix, test locally and submit a PR in the usual manner.
> 
> ### For merge team/dev lead
> * [ ]  Review recommendations.  If you agree with the recommendation to dismiss the alert, dismiss the alert.
> * [ ]  Remove the `ready for dev lead` label and close the issue
> * [ ]  If new issues are required to make code fixes, put the issue in "New Issue Approval" with "ready for dev lead" label
> * [ ]  Check off the Issue in [Create Issues for Analysis of CodeQL alerts #5060](https://github.com/hackforla/website/issues/5060)
> 
> ### Resources/Instructions
> * [GitHub CodeQL documentation](https://codeql.github.com/docs/codeql-overview/about-codeql/)
> * This issue is tracked in the spreadsheet [HfLA website: CodeQL scan alerts (issue #5060)](https://docs.google.com/spreadsheets/d/1B3R-fI8OW0LcYuwZICQZ2fB8sjlE3VsfyGIXoReNBIs/edit#gid=193401043) and [Create Issues for Analysis of CodeQL alerts #5060](https://github.com/hackforla/website/issues/5060)

**Comment by KyleA99** [comment]
i. Availability: 10/19/23 - 10/21/23
ii. ETA: Before meeting on Sunday 10/22/23

**Comment by KyleA99** [comment]
Dismiss as false positive

**Comment by roslynwythe** [comment]
Thank you @KyleA99.  This issue is closed as completed.
